packages/repositories.config

```diff
@@ -1,10 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <repositories>
-  <repository path="..\Octokit\packages.config" />
+  <repository path="../Octokit.Reactive/packages.config" />
```

Comment:
Not sure how this got introduced (I've seen VS do this from time to time, and it's not exactly repeatable). Are you able to revert these?

```
> # set this file to whatever is on master
> git checkout master -- packages/repositories.config
> # amend this comment
> git add .
> git commit --amend
> # force push to rewrite this PR
> git push origin misspelt-documentation --force
```